aliyun-ecs/src/main/java/org/jclouds/aliyun/ecs/features/ImageApi.java

```diff
@@ -52,15 +52,15 @@
 @Consumes(MediaType.APPLICATION_JSON)
 @RequestFilters(FormSign.class)
 @QueryParams(keys = {"Version", "Format", "SignatureVersion", "ServiceCode", "SignatureMethod"},
-        values = {"2014-05-26", "JSON", "1.0", "ecs", "HMAC-SHA1"})
+        values = { ECSServiceApiMetadata.DEFAULT_API_VERSION, "JSON", "1.0", "ecs", "HMAC-SHA1"})
```

Comment:
Use `"{" + Constants.PROPERTY_API_VERSION + "}"` instead for the version, so the value of the variable is used. This way instead of having a hardcoded value, it will pick the default one, or the one configured by users at runtime when creating the context via the `apiVersion` method.

Reply:
cool, sorry I forgot to address your previous comment on that and I left an intermediate solution